.travis.yml

```diff
@@ -32,20 +32,31 @@ install:
   - pip install -r rootfs/dev_requirements.txt
   - make -C docs/ deps
   - GLIDE_HOME=/home/travis/.glide make -C client bootstrap
+  - make prep-bintray-json
```

Comment:
I didn't realize it got called twice... where do you see that behavior documented (I couldn't find either case)?